<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings_window</class>
 <widget class="QWidget" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix2/resources/linux.svg</normaloff>:/new/prefix2/resources/linux.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="settings_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SETTINGS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="distribution_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Distribution:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="distribution_layout">
     <item>
      <spacer name="distribution_spacer_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="distribution_select">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>SUSE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ubuntu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fedora</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Arch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CentOS/RedHat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Debian</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="distribution_spacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This sets proper service names according to your distribution.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="install_label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Install:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="2">
      <widget class="QPushButton" name="install_dhcp">
       <property name="text">
        <string>DHCP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="install_exo">
       <property name="text">
        <string>exo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="install_phpmyadmin">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>PhpMyAdmin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="install_samba">
       <property name="text">
        <string>Samba</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="install_ftp">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>vsftpd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="install_nfs">
       <property name="text">
        <string>NFS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="install_apache">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Apache2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="install_mysql">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>MySQL</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="install_dns">
       <property name="text">
        <string>DNS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="install_libvirt">
       <property name="text">
        <string>Virtualization</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="install_backupmanager">
       <property name="text">
        <string>backup-manager</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="install_docker">
       <property name="text">
        <string>docker</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
